### PR TITLE
[DS-71] Feature/indeterminate loader

### PR DESCRIPTION
**Pull request by panvourtsis**
## Description

This PR introduce the new indeterminate loader needed for login page for now and maybe more for the future.

[Ticket](https://orfium.atlassian.net/secure/RapidBoard.jspa?rapidView=40&projectKey=DS&modal=detail&selectedIssue=DS-71)

## Screenshot

image

## Test Plan
snapshots

### CODE REVIEW

File: src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { rem } from 'polished';
+import { Theme } from 'theme';
+
+export const LoaderLine = () => (theme: Theme): SerializedStyles => css`
+  position: absolute;
+  opacity: 0.4;
+  background: ${theme.utils.getColor('lightCoolGray', 400)};
+  width: 150%;
+  height: ${rem(5)};
```
Comment (elektracodes) [inline src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx:10]: The height is 8px but here is 5px (if we translate the rem to px)

Comment (elektracodes) [reply src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx:10]: Also border-radius is missing

**Review by elektracodes — CHANGES_REQUESTED** [review]

**Review by mkarajohn — APPROVED** [review]

File: src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx (hunk 1 above)
Comment (panvourtsis) [reply src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx:10]: loader line in the design doesn't have a radius
image

Comment (elektracodes) [reply src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx]: The front should be with a radius. The current one is not the same as your screenshot.
Screenshot_2

**Review by elektracodes — APPROVED** [review]

**Pull request merged**